master/internal/api_workspace.go

```diff
@@ -86,8 +86,10 @@ func (a *apiServer) GetWorkspaceProjects(
 	if err != nil {
 		return nil, err
 	}
-	if _, err = a.GetWorkspaceByID(req.Id, *curUser, false); err != nil {
-		return nil, err
+	if req.Id != 0 {
```

Comment:
Could you modify the comment on ```GetWorkspaceRequest``` field ```id``` in ```proto/src/determined/api/v1/workspace.proto``` to document this new behavior and have it show up in the swagger docs

Reply:
You meant `GetWorkspaceProjectsRequest`, right? I added comment in `GetWorkspaceProjectsRequest`

Reply:
Yeah my bad.